.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Ensure end-of-file newline
+# Soft indents (2 spaces)
+# Trim trailing whitespace
```

Comment:
Is it normal in .editorconfig files to have comments that say the same thing as the config itself, and that are separated this much from the associated config lines? This seems like the sort of situation where the comments and code could get out of sync really easily.

I'm not sure the comments are necessary, except perhaps `# Unix-style newlines` which to a new coder might not be obvious from `end_of_line = lf` (although a new coder should read the editorconfig docs if they are uncertain about anything they see here, so perhaps even that comment isn't needed).

The `# top level config` and `# All files` comments could stay as subheadings.